README.md

```diff
@@ -35,6 +35,36 @@ RHEL/Fedora:
 sudo dnf install openssl-devel gtk3-devel cairo-devel alsa-lib-devel
 ```
 
+##### BSD [WIP]
+
+Similar to Linux, Druid defaults to GTK while providing an X11 backend as well.
+Furthermore, bindgen must be able to find LLVM through the expected environment variable.
+Only OpenBSD/amd64 has been tested so far.
+
+OpenBSD:
+```shell
+doas pkg_add gtk+3 cairo llvm
+export LIBCLANG_PATH=/usr/local/lib
+```
+
+In case rustc(1) fails building bigger crates
+```shell
+memory allocation of 2880 bytes failed
+error: could not compile `gtk`
+
+Caused by:
+  process didn't exit successfully: `rustc --crate-name gtk [...]` (signal: 6, SIGABRT: process abort signal)
+warning: build failed, waiting for other jobs to finish...
+```
+try increasing your user's maximum heap size:
+```shell
+ulimit -a | grep data
+data(kbytes)       1572864
+```
+```shell
+ulimit -d  $(( 2 * 1572864 ))
+```
```

Comment:
Could this be made into a one-liner? Any adverse side-effects should also be noted,

Reply:
> Could this be made into a one-liner? Any adverse side-effects should also be noted,

I folded them.

These [ulimits](https://man.openbsd.org/ksh.1#ulmit) are limited to the shell one sets them from, and its children, i.e. cargo(1) and rustc(1), of course.

I am not aware of bad side effects -- you just get to allocate twice as much heap data per process from this shell.
If processes reach that limit, they themselves get killed;  afterall, this is not Linux where OOM may kill any process.